### PR TITLE
Documentation/update summary main challenges

### PR DESCRIPTION
**Pull request by JessicaFornetti**
### Description

updates the final report, makes the summary of main challenges shorter and more concise. 

Fixes # (issue)

### Type of change

Please select the option that best describes the changes made:

- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [xx] Documentation update

### Changes

updates the final report, makes the summary of main challenges shorter and more concise.

### CODE REVIEW

File: Documentation/final-report/pages/technical_solution/machine_learning.tex [hunk 1]
```diff
@@ -770,12 +770,11 @@ \subsubsection{Summary of main challenges}
 
 The biggest initial
 hurdle was finding a labelling software to annotate our satellite images.
-Initially, we had settled on LabelImg, however it was no longer being supported
-on most of our operating systems and was crashing on launch. Instead, we turned
-towards Label Studio and the ML Backend extension, which we intended on using
+Initially, we turned towards Label Studio and the ML Backend extension,
```
Comment (1Solon) [inline Documentation/final-report/pages/technical_solution/machine_learning.tex:773]: 'we' is passive

File: Documentation/final-report/pages/technical_solution/machine_learning.tex [hunk 2]
```diff
@@ -770,12 +770,11 @@ \subsubsection{Summary of main challenges}
...
 hurdle was finding a labelling software to annotate our satellite images.
-Initially, we had settled on LabelImg, however it was no longer being supported
-on most of our operating systems and was crashing on launch. Instead, we turned
-towards Label Studio and the ML Backend extension, which we intended on using
+Initially, we turned towards Label Studio and the ML Backend extension, 
+which we intended on using
 for its active learning functionality that would have enabled using YOLO v5 to
 automatically label our training images, after only annotating a few images.
-After much unsuccessful troubleshooting, we decided to switch approaches to
+However, after much troubleshooting, we decided to switch approaches to
```
Comment (1Solon) [inline Documentation/final-report/pages/technical_solution/machine_learning.tex:777]: again 'we'

File: Documentation/final-report/pages/technical_solution/machine_learning.tex [hunk 3]
```diff
@@ -789,33 +788,26 @@ \subsubsection{Summary of main challenges}
 
 Another challenge referred to the low resolution of the satellite images,
 heavily contributing to the high misclassification rate in the early beginnings of
-model training. We managed to stabilised that through proper model choosing and
+model training. We managed to stabilise it through proper model choosing and
```
Comment (1Solon) [inline Documentation/final-report/pages/technical_solution/machine_learning.tex:791]: 'we'

File: Documentation/final-report/pages/technical_solution/machine_learning.tex [hunk 4]
```diff
@@ -828,20 +820,14 @@ \subsubsection{Summary of main challenges}
 Originally 4 cases were considered, cars parked horizontally in a row or in a
 column, cars parked vertically in a column and cars parked vertically side by
 side. However, horizontally parked in a column and vertically parked side by side
-were removed as they led to too many misclassifications, namely the vertically
-parked side by side lead to adding many additional cars in driveways in
-residential areas.
-Additionally the condition on the angle alignment was refined and added to ensure
-the detection of spots only between cars properly aligned.
+were removed as they led to too many misclassifications.\\
```
Comment (1Solon) [inline Documentation/final-report/pages/technical_solution/machine_learning.tex:823]: Get rid of the `\\`

File: Documentation/final-report/pages/technical_solution/machine_learning.tex [hunk 5]
```diff
@@ -828,20 +820,14 @@ \subsubsection{Summary of main challenges}
...
 
 Another major issue was the selection and accurate labelling of the test sets to
 evaluate each subpart of our model.\\
```
Comment (1Solon) [inline Documentation/final-report/pages/technical_solution/machine_learning.tex:826]: Remove the `\\`

File: Documentation/final-report/pages/technical_solution/machine_learning.tex [hunk 6]
```diff
@@ -828,20 +820,14 @@ \subsubsection{Summary of main challenges}
...
 Week 11 marked the beginning of the unoccupied parking detection evaluation. 
 Labelling the test set accurately was a significant challenge, multiple different 
 phases of relabelling and improvements occurred, to ensure the maximum overlap 
-between the true labels and the model's predictions, achieving a high enough IoU.
-The images were originally annotated using rotated bounding boxes, however due to the fact that
-the rotation cannot be exported in Label Studio, they were subsequently relabelled without rotation.\\
```
Comment (1Solon) [inline Documentation/final-report/pages/technical_solution/machine_learning.tex:844]: Remove the `\\`

File: Documentation/final-report/pages/technical_solution/machine_learning.tex [hunk 7]
```diff
@@ -828,20 +820,14 @@ \subsubsection{Summary of main challenges}
...
 
 Another major issue was the selection and accurate labelling of the test sets to
 evaluate each subpart of our model.\\
 Week 11 marked the beginning of the unoccupied parking detection evaluation. 
 Labelling the test set accurately was a significant challenge, multiple different 
 phases of relabelling and improvements occurred, to ensure the maximum overlap 
-between the true labels and the model's predictions, achieving a high enough IoU.
-The images were originally annotated using rotated bounding boxes, however due to the fact that
-the rotation cannot be exported in Label Studio, they were subsequently relabelled without rotation.\\ 
+between the true labels and the model's predictions.\\
```
Comment (1Solon) [inline Documentation/final-report/pages/technical_solution/machine_learning.tex:830]: Remove the `\\`

**Review by 1Solon — CHANGES_REQUESTED** [review]
Please fix the above

File: Documentation/final-report/pages/technical_solution/machine_learning.tex (hunk 1 above)
Comment (anaisbl) [inline Documentation/final-report/pages/technical_solution/machine_learning.tex:773]: We started with Label Studio, a popular labelling software with their integrated ML backend extension

File: Documentation/final-report/pages/technical_solution/machine_learning.tex [hunk 8]
```diff
@@ -770,12 +770,11 @@ \subsubsection{Summary of main challenges}
 
 The biggest initial
 hurdle was finding a labelling software to annotate our satellite images.
-Initially, we had settled on LabelImg, however it was no longer being supported
-on most of our operating systems and was crashing on launch. Instead, we turned
-towards Label Studio and the ML Backend extension, which we intended on using
+Initially, we turned towards Label Studio and the ML Backend extension, 
+which we intended on using
```
Comment (anaisbl) [inline Documentation/final-report/pages/technical_solution/machine_learning.tex:774]: remove "which we intended on using...." until "that would have enabled"

File: Documentation/final-report/pages/technical_solution/machine_learning.tex [hunk 9]
```diff
@@ -770,12 +770,11 @@ \subsubsection{Summary of main challenges}
...
 for its active learning functionality that would have enabled using YOLO v5 to
 automatically label our training images, after only annotating a few images.
-After much unsuccessful troubleshooting, we decided to switch approaches to
+However, after much troubleshooting, we decided to switch approaches to
 manually label the images and subsequently train the YOLO model on those images.
 
 The next challenge related to training the YOLO model to identify cars on the
```
Comment (anaisbl) [inline Documentation/final-report/pages/technical_solution/machine_learning.tex:780]: replaced related with concerned

**Review by anaisbl — CHANGES_REQUESTED** [review]

**Review by 1Solon — APPROVED** [review]

**Review by anaisbl — APPROVED** [review]

**Pull request merged**